Buildings/Buildings.csv

```diff
@@ -199,7 +199,6 @@ building_id,building_code,parent_code,building_name,alternate_building_names,lat
 149,KSW,,195 King Street West,,,,1
 150,HSC,,Huntsville Summit Centre,,45.32211,-79.20695,1
 151,UAE,,United Arab Emirates Dubai,,,,1
-152,WSS,,"6 Wellington Street, Stratford",,43.370474,-80.982231,1
```

Comment:
@jmtjendra instead of deleting 152, you can change the "1" at the end of the line to a "0".